rotkehlchen/assets/resolver.py

```diff
@@ -82,14 +98,26 @@ def resolve_asset(
     def get_asset_type(identifier: str) -> AssetType:
         # TODO: This is ugly here but is here to avoid a cyclic import in the Assets file
         # Couldn't find a reorg that solves this cyclic import
+        from rotkehlchen.constants.assets import CONSTANT_ASSETS  # pylint: disable=import-outside-toplevel  # isort:skip  # noqa: E501
         from rotkehlchen.globaldb.handler import GlobalDBHandler   # pylint: disable=import-outside-toplevel  # isort:skip  # noqa: E501
 
         instance = AssetResolver()
         cached_data = instance.types_cache.get(identifier)
         if cached_data is not None:
             return cached_data
 
-        asset_type = GlobalDBHandler().get_asset_type(identifier)
+        try:
+            asset_type = GlobalDBHandler().get_asset_type(identifier)
+        except UnknownAsset:
```

Comment:
same here

Reply:
`get_asset_type` doesn't raise `WrongAssetType` https://github.com/rotki/rotki/blob/0fa9a7cc2dee688db0621750a31b2c6c49c31108/rotkehlchen/globaldb/handler.py#L1786

Reply:
okay fair enough. This one can be ignored then.